requirements/environment.yml

```diff
@@ -4,6 +4,12 @@ channels:
   - defaults
 dependencies:
   - python=3.8
+  - jupyterlab
```

Comment:
Yep, I agree! My experience with conda has been mixed as well. But since the documentation strongly recommends using conda for GeoPandas (see screenshot below from https://geopandas.org/install.html#installing-with-pip), maybe we should just install GeoPandas with conda and then everything else with pip? That should work, not sure it's more stable than just using pip. We definitely don't want anybody to need to compile C libraries themselves.

Also, has the master environment been tested from scratch? Does geopandas work with no problem there, I haven't had time to do it yet. If it does I am also happy to just keep it all pip for now.

![Screenshot 2021-02-12 at 18 25 35](https://user-images.githubusercontent.com/75615911/107807235-ba4f1e00-6d5f-11eb-91c3-042e45177696.png)

Reply:
I tested the master branch installation from scratch, and it finished without errors. Doing this means all the packages including GeoPandas, fiona, etc are retrieved from the pip database, so I think the dependencies are fine with pip. I think some of the dependencies already exist in on JASMIN (e.g. `gdal-config` utility is there, which is a fiona dependency).

Therefore I think we can just have all dependencies in the `dev-requirements.txt` file.

Reply:
Great, thanks for testing it! I moved the dependencies to pip (via `dev-requirements.txt`) now.